pages/en/advanced/bp-sidecar.mdx

```diff
@@ -7,17 +7,17 @@ export default Page({ title: "Block Producer Sidecar" });
 <br></br>
 
 <Alert kind="info">
-<p>
-<b>IMPORTANT UPDATE:</b> This will affect the submission of uptime data and the Performance Score results for receiving a delegation.
-</p>
-<br></br>
-<p>
-The current sidecar tracking system will be phased out after the current cycle (Cycle 6) and be replaced by the SNARK-work-based uptime tracking system. Until further notice, while transitioning to the new system, <b>continue running the <u>sidecar</u> AND <u>SNARK-based tracking system</u> on the SAME node.</b> 
-</p>
+
+**PLEASE READ:** This affects the submission of uptime data and the Performance Score results for receiving a delegation.
 <br></br>
-<p>
-Please follow the instructions in <a href="https://discord.com/channels/484437221055922177/808895957978447882/958384520464330812">this post here</a> and ask any questions in the <a href="https://discord.gg/mXXKHMDU">#delegation-program channel on Discord</a> while following along for the latest updates.
-</p>
+
+The current sidecar tracking system is being phased out to be replaced by the SNARK-work-based uptime tracking system. Until further notice, <u>please continue running the sidecar AND SNARK-based tracking system on the SAME node.</u>
+
+- <DocLink copy="Instructions for the sidecar tracking system" url="/advanced/bp-sidecar" />
+- <DocLink copy="Instructions for the SNARK-work-based uptime system" url="/advanced/uptime-system" />
+
+Please follow the latest updates and post questions in the [#delegation-program channel](https://discord.gg/gEpE9pdC) on Discord.
```

Comment:
`https://discord.gg/gEpE9pdC`

Is this discord invite expired? Doesn't seem to work for me :/